experimenter/experimenter/experiments/api/v6/views.py

```diff
@@ -37,3 +48,55 @@ class NimbusExperimentFirstRunViewSet(NimbusExperimentViewSet):
         .filter(is_first_run=True)
         .order_by("slug")
     )
+
+
+class NimbusFmlDiagnosticsApiSchema(AutoSchema):  # pragma: no cover
+    def get_operation(self, path, method):
+        operation = super().get_operation(path, method)
+        operation["parameters"] = [
+            {
+                "name": "application",
+                "in": "query",
+                "required": True,
+                "schema": {"type": "string"},
+            },
+            {
+                "name": "channel",
+                "in": "query",
+                "required": True,
+                "schema": {"type": "string"},
+            },
+            {
+                "name": "blob",
+                "in": "query",
+                "required": True,
+                "schema": {"type": "string"},
+            },
+        ]
+        return operation
+
+
+class NimbusFmlDiagnosticsApi(APIView):
+    def post(self, request):  # pragma: no cover
+        self.schema = NimbusFmlDiagnosticsApiSchema
+        application = request.POST.get("application")
+        channel = request.POST.get("channel")
+        request.POST.get("blob")
```

Comment:
Will this be eventually:
```suggestion
        blob = request.POST.get("blob")
```

Reply:
Gah! Good catch :)